src/ADAL.PCL.Desktop/PlatformInformation.cs

```diff
@@ -79,12 +81,41 @@ public override string GetOperatingSystem()
             return Environment.OSVersion.ToString();
         }
 
+        public override string GetDeviceId()
+        {
+            string DeviceId = "";
+            ManagementObjectSearcher MOS = new ManagementObjectSearcher("SELECT DeviceID FROM Win32_Processor");
```

Comment:
managementObjectSearcher

---
In reply to: [82043453](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/494#discussion_r82043453) [](ancestors = 82043453)